### ISSUE TITLE
Pytorch official syncbn in pytorch 1.1 brach

### ISSUE THREAD

**Issue by Yuyn1234** [HRNet/HRNet-Semantic-Segmentation#173]
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function

import os
import logging
import functools

import numpy as np

import torch
import torch.nn as nn
import torch._utils
import torch.nn.functional as F

BatchNorm2d = nn.BatchNorm2d
BN_MOMENTUM = 0.01

The codes are shown above. It seems that you do not utilize syncbn.
